tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -314,32 +385,99 @@ private static void GenerateMainProject(ApiMetadata api, string directory, HashS
                 propertyGroup.Add(new XElement("CodeAnalysisRuleSet", "..\\..\\..\\grpc.ruleset"));
             }
             var dependenciesElement = CreateDependenciesElement(api.Id, dependencies, api.IsReleaseVersion, testProject: false, apiNames: apiNames);
+            if (api.HasAnalyzers)
+            {
+                void AddPackFile(string includePath, string packagePath)
+                {
+                    dependenciesElement.Add(
+                        new XElement("None",
+                            new XAttribute("Include", includePath),
+                            new XAttribute("Pack", "true"),
+                            new XAttribute("PackagePath", packagePath),
+                            new XAttribute("Visible", "false")));
+                }
+
+                AddPackFile(
+                    $@"..\{api.Id}.Analyzers\bin\$(Configuration)\{AnalyzersTargetFramework}\{api.Id}.Analyzers.dll",
+                    "analyzers/dotnet/cs");
```

Comment:
Does it matter that the paths here and above are using different separators?

Reply:
This is mimicking what you get out of the box when you create a new analyzer in VS, so no I don't think so. That said, I did hit a bug with the wrong slashes in the .sln file on Linux in the PR. But this seems to be fine on the build machines, so I'll leave as is.